pyasic/rpc/luxminer.py

```diff
@@ -844,7 +841,7 @@ async def voltageset(self, board_n: int, voltage: float) -> dict:
         """
         return await self.send_privileged_command("voltageset", board_n, voltage)
 
-    async def upgraderun(self):
+    async def updaterun(self) -> dict:
```

Comment:
Renamed this since command is [updaterun](https://docs.luxor.tech/firmware/api/luxminer/updaterun), not upgraderun